linode/manager
# M3-5965: Add Generic Support Text Link Replacement to Notices

## Description 📝

**What does this PR do?**
Links the words "open a support ticket" in error notices to the support page with an open ticket modal. We previously did this for the Account Limit error only; this PR intends to link "open a support ticket" for all errors intercepted from the API with these key words that are not field errors, and replace "open a support ticket" anywhere it exists the error message reason.

## Preview 📷
![Screen Shot 2022-10-27 at 1 47 13 PM](https://user-images.githubusercontent.com/114685994/198397719-b84b9b54-6bd0-4258-843b-8a75c32e2758.jpg)
![Screen Shot 2022-10-27 at 2 01 55 PM](https://user-images.githubusercontent.com/114685994/198397448-649b9aa1-47bd-42cf-a6f7-528cc61b1c5b.jpg)
![Screen Shot 2022-10-27 at 2 02 31 PM](https://user-images.githubusercontent.com/114685994/198397483-fd9ffc6d-8484-44ce-9541-5d4c3c3ffcfe.jpg)

## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
Tests from the [previous PR](https://github.com/linode/manager/pull/8566) still apply, but this can also be tested in `serviceHandlers.ts` by commenting out the payload, linode, and successful mocked response in `rest.post('*/instances', async (req, res, ctx)` and adding the error response below.  Test the following error messages (one at a time) when creating a linode with MSW:

```
    return res(
      ctx.status(400),
      ctx.json({
        errors: [
          // {
          //   reason: 'Open a support ticket for this issue.',
          // },
          // {
          //   reason:
          //     'Additional verification is required to add this service. Please open a support ticket.',
          // },
          // { reason: 'Please open a Support Ticket describing your requirement'}
        ],
      })
    );
```
